drape/gl_functions.cpp

```diff
@@ -255,8 +255,17 @@ void GLFunctions::Init(dp::ApiVersion apiVersion)
     glUnmapBufferFn = &::glUnmapBuffer;
 
 #elif defined(OMIM_OS_LINUX)
-
-    CHECK(false, ("OpenGLES2 is not yet supported"));
+    void *libhandle = dlopen("libGL.so.1", RTLD_NOW);
```

Comment:
1. If the execution reaches this point, then it is very unlikely that we gain anything from trying to do a lazy resolution, as resolution is going to happen super soon anyway. But if you see value in it then I can change it.
2. `dlclose` would decrement the reference counter to the `.so`, and would potentially unload functions that we will have to keep using till the end of the application's runtime.

Reply:
1. Does RTLD_NOW resolve _all_ functions? If yes, then does LAZY resolve only the requested ones? We need only a few of them.

2. We prefer to properly clean up all requested resources, but it may be complicated in this case. Question: is it possible to use `eglGetProcAddress` here instead of dlopen?

Reply:
3. And another question: did you test all 3 cases here? 1) OpenGL ES 2; 2) OpenGL ES 3; 3) Desktop OpenGL?

Reply:
1. In any case, I have chaned it back to LAZY.

Reply:
2.  Well, the first versions of this PR used `eglGetProcAddress` because it linked to EGL, which was also a new dependency, and that (new dependencies) was commented as a source of potential problems and impacts. So what has changed?

Reply:
Now I remember, thanks for the reminder. I've re-read the comments, and looks like I was wrong, and confused EGL with GLES 🤦 that time. The platform-correct way likely would be to link EGL on Linux and use its methods where necessary. But the existing approach doesn't create additional build dependency so it also makes good sense.
Maybe a comment mentioning that an "appropriate" way to get these addresses would be to use eglGetProcAddress and EGL would be enough ATM.